lib/bible_ref/languages/russian.rb

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+require_relative 'base'
+
+module BibleRef
+  module Languages
+    class Russian < Base
+      # Is it a single chapter book?
+      def has_single_chapter?(reference)
+        matches = [/^ob/, /^(jud|jd|jude)\b/, /^2 ?jo?h?n/, /^3 ?jo?h?n/, /^(philem|phm|pm)/]
```

Comment:
oops, I forgot to remove this line, just a moment, I'll fix this and update the PR.